src/app/book/edit/edit.component.ts

```diff
@@ -0,0 +1,64 @@
+import {Component, Inject} from '@angular/core';
+import {MD_DIALOG_DATA, MdDialogRef, MdSnackBar} from '@angular/material';
+import {BookService} from '../../services/book.service';
+import {Book} from '../../models/book.model';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-edit-book',
+  templateUrl: './edit.component.html',
+  styleUrls: ['./edit.component.css']
+})
+export class EditComponent {
+
+  bookForm: FormGroup;
+
+  minPrice = 1;
```

Comment:
This is the second time I'm seeing these requirements. Maybe we can add them as part of the Book class as static members and reuse them?